Start service rules for schools in Register ECTs (#727)
DFE-Digital/register-early-career-teachers: documentation/rect-school-admins.md

* school induction coordinators

## Getting access to the service

For a school admin to get access to the service, they will need to use [DfE Sign in](https://services.signin.education.gov.uk/).

DfE Sign-in is how schools and other education organisations access DfE online services. We've decided to use it for Register ECTs because:
- schools use it for other services, and were getting confused in ECF1
- in research, most school users had an understanding of DfE Sign in
- access to the service historically worked through using info from Get information about schools, but many schools weren't aware of this
- it still makes sure the person accessing the service actually works for the school by going through an approval process

You can read more about why we chose to do this in [our design history](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/exploring-using-dfe-sign-in/).

To get access to the service, a school will need to request for approval. This is checked and actioned by whoever at their school has approval permissions. Once this is done, they will be able to sign into the service and view the records for their school.

## Accessing the service

> We will not limit the number of users per school. This means multiple people can get access, unlike in ECF1.

Reviewer: ```suggestion
We will not limit the number of users per school. This means multiple people can get access, unlike in ECF1. Schools will manage the users who can access the service for their school themselves via DfE Sign In.
```

You can read more about [why we made this decision in our design history](https://teacher-cpd.design-history.education.gov.uk/ecf-v2/allowing-multiple-school-accounts/). 

## Registering an ECT

### Finding an ECT's record in the Teaching Record System

> 

Reviewer: ```suggestion
When a school registers an ECT, we need to find the ECT's TRS record so that we can check they are a real person and so that we can determine their eligibility for training. 
```

At this stage, when we check the TRS for the existence of the ECT's record, we also need to make sure:
- the ECT has not completed induction already
- the ECT is not exempt from induction

Whilst an ECT needs qualified teaching status in order to be eligible for funding for training, we do allow them to be registered in advance without it. 

This is because we know schools might want to register an ECT before they actually start working in a school. This might mean that ECT doesn't always have QTS before they are registered for training.

### Transferring in an early career teacher

We still need to define what happens here more clearly.

For now, we know when an ECT is registered by a new school to the school they're already registered with, and that ECT is 'in progress', we should start the transfers journey.

### Confirming or correcting an ECT's name

> Once we've decided the ECT is eligible to be registered for training, we play back the name from the TRA's Teaching Record System to the user. This is so we can show them it's linked to their teacher record and that they must be intending to register that person.

Reviewer: ```suggestion
Once we've decided the ECT can be added to the service, we play back the name from the TRA's Teaching Record System to the user. This is so we can show them it's linked to their teacher record and that they must be intending to register that person.
```
